invenio_administration/marshmallow_utils.py

```diff
@@ -55,8 +56,10 @@ def jsonify_schema(schema):
 
     for field, field_type in schema.fields.items():
         is_links = isinstance(field_type, invenio_fields.links.Links)
+        # skip `fields.Method`
+        is_method = isinstance(field_type, fields.Method)
 
-        if is_links:
+        if is_links or is_method:
```

Comment:
wondering if we want to have `dump_only` fields in the jsonschema or we could also use `is_read_only` in the logic (see #65)